nucleus/payara-modules/requesttracing-core/src/main/java/fish/payara/nucleus/requesttracing/store/RequestTraceStoreFactory.java

```diff
@@ -86,8 +89,8 @@ public static RequestTraceStoreInterface getStore(Events events, boolean reservo
         // Get a clustered store if possible
         ClusteredStore clusteredStore = Globals.getDefaultHabitat().getService(ClusteredStore.class);
         if (clusteredStore != null && clusteredStore.isEnabled()) {
-            MultiMap<String, RequestTrace> store = (MultiMap) clusteredStore.getMultiMap(storeName);
-            return new ClusteredRequestTraceStore(store, clusteredStore.getInstanceId(), strategy);
+            Map<UUID, RequestTrace> store = (Map) clusteredStore.getMap(storeName);
+            return new ClusteredRequestTraceStore(store, strategy);
```

Comment:
See comment on `ClusteredRequestTraceStore` on the change from `MultiMap` to `Map`